File: Assignments/Assignment_19/p7.c
#include<stdio.h>
void main(){

        int size,count=0;
        printf("Enter the size of array :\n");
        scanf("%d",&size);
        int arr[size],arr1[size];
        for(int i=0;i<size;i++){

                int temp,sum,fact,fact1=0;
                printf("Enter the arr element %d : ",i+1);
                scanf("%d",&arr[i]);
                temp=arr[i];
                while(temp>0){

                        fact=1;
                        sum=temp%10;
                        temp/=10;
                        for(int j=1;j<=sum;j++){

                                fact*=j;
                        }fact1+=fact;
                }arr1[i]=fact1;
                if((arr[i]==arr1[i])&&(arr[i]!=1)) count++;
        }if(count>0){

                printf("Array contains %d strong number",count);
        }else{

                printf("Array contains no strong numbers");
        }
}

File: Flutter/Assignment/For Loop-2/p7.dart
import "Dart:io";
void main(){
	print("Enter a number:");
	int? x=int.parse(stdin.readLineSync()!);
	List num=[];
	for(int i=0;i<x;i++){
		int? element=int.parse(stdin.readLineSync()!);
		num.add(element);
	}
	int count=0;
	for(int i=0;i<x;i++){
		if(num[i]>50){
			count++;
		}	
	}
	print("Count of numbers > 50: $count");
}
